config/owasp/suppressions.xml

```diff
@@ -1470,10 +1470,24 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
-   file name: spring-webmvc-5.0.8.RELEASE.jar
+   file name: spring-webmvc-5.2.2.RELEASE.jar
    ]]></notes>
```

Comment:
remove this CVE as after upgrading Spring version , this is entry is not required.

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/240/commits/9a800a1c26dc64dead6d101103e08710580fc129